server/schemas/schemaModel.js

```diff
@@ -1,10 +1,53 @@
 var mongoose = require("mongoose");
-
+var bluebird = require("bluebird");
+var helpers = require("../config/helpers");
 var Schema = mongoose.Schema;
-
 var schemaSchema = new Schema({
```

Comment:
Maybe formValidator or formConstraints ? (ex. formValidatorSchema or formConstraintsSchema) This might provide more clarity since Rachel's front end is basically a form and we need to make sure it is clear that this is not the form itself, but the thing that checks the form is correct.